desktop/surfaces/gui/src/styles.css

```
@@ -42,99 +52,255 @@
-  padding: 10px 16px;
+.contacts-search input::placeholder {
+  color: var(--muted);
+}
+
+.contacts-visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)